client/src/sim.rs

```diff
@@ -163,12 +166,29 @@ impl Sim {
                 self.step = Some(msg.step);
                 let my_id = self.params.as_ref().map(|params| params.character_id);
                 let mut my_position = None;
+                let mut my_velocity = None;
                 for &(id, new_pos) in &msg.positions {
                     if Some(id) == my_id {
                         my_position = Some(new_pos)
                     }
                     self.update_position(id, new_pos);
                 }
+                for &(id, velocity) in &msg.character_velocities {
+                    if Some(id) == my_id {
+                        my_velocity = Some(velocity)
+                    }
+                    match self.entity_ids.get(&id) {
+                        None => debug!(%id, "character orientation update for unknown entity"),
+                        Some(&entity) => match self.world.get::<&mut Character>(entity) {
+                            Ok(mut ch) => {
+                                ch.velocity = velocity;
+                            }
+                            Err(e) => {
+                                error!(%id, "character orientation update for non-character entity {}", e)
```

Comment:
`e` will display as something like "missing component" or "no such entity", so this wording is pretty strange.

Reply:
This was copied from the existing `Err(e) => error!(%id, "position update for unpositioned entity {}", e),` in `update_position(...)`. I'll update both.